pytheus/backends.py

```diff
@@ -0,0 +1,17 @@
+from threading import Lock
+
+
+class LockedValue:
```

Comment:
for experimenting I created only this one, but work will be needed on the pluggable system